gspread-project/push_data.py

```diff
@@ -1,15 +1,22 @@
+import os
 import gspread
 import pandas as pd
-
+import logging
 import warnings
 warnings.filterwarnings('ignore')
 
+logging.basicConfig(level=logging.INFO)
+
 def service_file(file_path: str) -> gspread.client:
     """
     this function load google service account file
     """
-    gc = gspread.service_account(filename=file_path)
-    print("service file loaded successfully")
+    # config = dotenv_values()
```

Comment:
If this is your final code, you should remove redundant comments that has no particular use.